src/main/java/hudson/tasks/junit/StdioRetention.java

```diff
@@ -0,0 +1,43 @@
+package hudson.tasks.junit;
+
+import java.util.Locale;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum StdioRetention {
+
+    ALL("All"),
+    FAILED("Failed tests only"),
+    NONE("None");
```

Comment:
These shouldn't be hardcoded in english, they need to use a messages class, see https://www.jenkins.io/doc/developer/internationalization/i18n-source-code/

Reply:
Thanks, done. I've also added help info for the `stdioRetention` option and added a deprecation notice for `keepLongStdio`. I've only added/altered English variants though. Do we need to add/alter German, Japanese, and Chinese info in this PR as well, or can that be addressed as a follow-up?

Reply:
no need to add them if you aren't a native speaker.

the normal convention is if you change an english localised string you clear out the translations.
if you remove an english localised string you remove all translations